website/pages/en/querying/distributed-systems.mdx

```diff
@@ -89,36 +90,38 @@ async function getDomainNames() {
 
   // The first query will get the first page of results and also get the block
   // hash so that the remainder of the queries are consistent with the first.
-  const listDomainsQuery = `
+  const listDomainsQuery = /* GraphQL */ `
```

Comment:
it always better adds special comment `/* GraphQL */` for graphql documents, so prettier will prettify them and graphql-eslint could validate even in markdown